server/src/domain/media/media.service.spec.ts

```diff
@@ -129,6 +129,22 @@ describe(MediaService.name, () => {
     });
   });
 
+  describe('handleGenerateThumbhashThumbnail', () => {
+    it('should skip thumbhash generate if resize path is missing', async () => {
+      assetMock.getByIds.mockResolvedValue([assetEntityStub.noResizePath]);
+      await sut.handleGenerateThumbhashThumbnail({ id: assetEntityStub.noResizePath.id });
+      expect(mediaMock.generateThumbhash).not.toHaveBeenCalled();
+    });
+
+    it('should generate a thumbhash', async () => {
+      assetMock.getByIds.mockResolvedValue([assetEntityStub.image]);
+      await sut.handleGenerateThumbhashThumbnail({ id: assetEntityStub.image.id });
+
+      expect(mediaMock.generateThumbhash).toHaveBeenCalledWith('/uploads/user-id/thumbs/path.ext');
+      expect(assetMock.save).toHaveBeenCalledWith({ id: 'asset-id', thumbhash: undefined });
```

Comment:
If you mock a response to `mediaMock.generateThumbhash.mockResolvedValue(Buffer.from('a thumbhash', 'utf8'))` you can assert the thumhash is saved in this call. Saving an undefined thumbhash seems like an off assertion. You can also put the `mockResolvedValue(...)` in the media.repository.mock.ts file and call it a "default response value", which is also fine.